api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -222,6 +222,15 @@ private User initializeUserIfNeeded() {
       // Service accounts don't need further initialization.
       return user;
     }
+
+
+    if (user != null &&
+        (user.getTwoFactorEnabled() == null ||
+            !(user.getTwoFactorEnabled()))) {
```

Comment:
I always have a hard time with multiple booleans that check both positive and negative conditions in the same check. For this kind of thing, I really like java's optional class. I think this makes the code read more clearly. 
```
Boolean twoFactorEnabled = Optional.ofNullable(user.getTwoFactorEnabled()).orElse(false);
if (!twoFactorEnabled) { ... }
```